src/style.css

```diff
@@ -0,0 +1,168 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+header {
+  padding: 0 50px;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+ul {
+  list-style: none;
+  display: flex;
+}
+
+li {
+  margin-left: 20px;
+  display: flex;
+}
+
+li:first-child {
+  margin: 0;
+}
+
+a {
+  font-family: "Roboto", sans-serif;
```

Comment:
`color` and `font-family` are inherited.
Better to put them into `body` and you don't need to duplicate them.